packages/application-components/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@commercetools-frontend/application-components",
+  "version": "7.0.0",
+  "description": "Generic components for building Merchant Center applications",
+  "bugs": "https://github.com/commercetools/merchant-center-application-kit/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/merchant-center-application-kit.git"
+  },
+  "keywords": [
+    "javascript",
+    "frontend",
+    "react",
+    "toolkit"
+  ],
+  "license": "MIT",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/application-components.cjs.js",
+  "module": "dist/application-components.es.js",
+  "files": [
+    "dist",
+    "materials",
```

Comment:
Nitpick: should we specify the single files, like in the ui-kit?

Reply:
yep, good call